.github/workflows/markdown-linter.yml

```diff
@@ -1,6 +1,6 @@
-name: "markdown-linter"
+name: markdown-linter
 on:
-  push:
+  workflow_call:
```

Comment:
Doesn't this mean that the linter will no be run on PRs and stuff? if we want to use this, we should also make the `lint` workflow call it, no?